src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -72,6 +73,10 @@ List all partitions in the cluster.
 List all partitions in the cluster, filtering for topic foo and bar.
   rpk cluster partitions list foo bar
 
+List partitions which replicas are assigned on specific brokers.
+
+  rpk cluster partitions list foo --brokers 1,2
```

Comment:
What do you think of this to make sure that we document `--brokers` being an inclusive filter?
```suggestion
List partitions in which replicas are assigned to brokers 1 and 2.
  rpk cluster partitions list foo --brokers 1,2
```

Reply:
+1, thank you!